index.html

```diff
@@ -1,17 +1,23 @@
 ---
-layout: index
+layout: base
```

Comment:
This is resulting in OCP branding being applied to the index page, and I prefer the simpler layout of a basic list.

![logonoff co_oauth-templates_](https://github.com/user-attachments/assets/a2339039-7658-4a18-a079-c3945a58bcce)